code/game/machinery/machinery.dm

```diff
@@ -130,6 +130,10 @@ Class Procs:
 		dropContents()
 	return ..()
 
+/obj/machinery/attackby(obj/item/weapon/W, mob/user, params)
+	user.changeNext_move(CLICK_CD_MELEE)
+	..()

```

Comment:
so we don't have to add the same click cooldown line for every child of machinery.